### DISCUSSION TITLE
PR Review Feedback box is too small

### DISCUSSION THREAD

**Discussion by ralgozino** [community/community#27437] [Pull Requests]
Hello!

Lately I had several moments where I wished I could have a bigger input to write PR review feedback. You know, the popup that appears when you click "Review Changes" as a Reviewer in a PR:
<img width="671" alt="Screenshot 2022-07-28 at 10 18 22" src="https://user-images.githubusercontent.com/6362698/181457055-53f1a1d9-e385-408e-9fe7-e8c69184b969.png">

I guess the current sizing is OK for most of the cases, but when you want to give a little more structured feedback, writing in that little input field is uncomfortable.

I think one nice option would be to have a button in the dialog that makes the popup an almost-full-screen modal dialog.

**Comment by jfullstackdev** [comment]
you can simply adjust that, bottom right corner, drag it

![image](https://user-images.githubusercontent.com/47092464/181508350-b51f2af1-c460-45b3-bb38-bf5989a6f446.png)

**Comment by ralgozino** [comment]
That is a browser-dependant feature, isn't it? In any way, that method has 2 downsides:
1. It lets you increase the size of the input **vertically** and not horizontally, at least in Chrome.
2. The dropdown is still anchored at the right-top part of the screen. Which makes it a little uncomfortable for non-small quantities of feedback.
